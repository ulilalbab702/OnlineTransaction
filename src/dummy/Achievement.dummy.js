export const Achievement = [
    {
        "Code": "FM-JBYPM",
        "Name": "FMC Jembayan PAMA",
        "Percent": 50,
        "BackLogOpen": 1,
        "BackLogClose": 1,
        "Priority1": 1,
        "Priority2": 0,
        "Priority3": 1,
        "Priority4": 0
    }
]