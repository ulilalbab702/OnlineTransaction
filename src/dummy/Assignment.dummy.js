export const listOfMechanicDummy = [
    {
      "Username": "string",
      "FirstName": "string",
      "LastName": "string",
      "Email": "string",
      "MechanicUT": true,
      "Jobs": [
        {
          "JobType": "string",
          "Total": 0
        }
      ]
    }
  ]
export const jobSummaryDummy = {
  "Date": "2020-06-04T16:18:05.5472055",
  "CreatedBy": "AGUS LISANTO",
  "AssignedBy": "AGUS LISANTO",
  "MechanicLeader": {
      "UserName": "82107038@unitedtractors.com",
      "Name": "SUPRIYADI SUPRIYADI"
  },
  "Mechanics": [],
  "HandoverDate": null,
  "HandoverCreatedBy": null,
  "HandoverAssignedBy": null,
  "HandoverMechanicLeader": null,
  "HandoverMechanics": null,
  "Job": {
      "WoId": "8ff577d1-2911-4bbd-9d55-ffad4fe76fc5",
      "WoNumber": "71018650",
      "JobType": "INS",
      "UnitModel": "HD465-7",
      "SerialNumber": "15163",
      "UnitCode": "HT3022",
      "EquipmentNumber": "71506",
      "IsHandover": false,
      "Id": "a162bd06-6aca-499a-9f97-a5839b8a0612",
      "CreatedBy": "80107002@unitedtractors.com",
      "CreatedDate": "2020-06-04T16:18:05.5472055",
      "UpdatedBy": "80107002@unitedtractors.com",
      "UpdatedDate": "2020-06-04T16:18:05.5472055",
      "IsActive": true
  }
}