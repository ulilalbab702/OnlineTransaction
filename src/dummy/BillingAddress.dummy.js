export const dataBillingAddressDummy = [
    {
        "fullName": "Test Integrasi",
        "contactNumber": "088122345",
        "email": "testintegrasi@unitedtractors.com",
        "position": "string",
        "addressLabel": "Kantor PT Global Service Indonesia",
        "city": "Jakarta",
        "address": "Jl Raya kaliabang Tengah, Ruko San Jaya, Blok A5 No 9,\r\nSebalah Indomaret",
        "postalCode": "13920",
        "locationLatitude": "3.669276961596295",
        "locationLongitude": "102.4003031491451",
        "isPrimary": true
    }]