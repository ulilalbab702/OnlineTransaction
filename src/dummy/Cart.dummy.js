export const cartDummyList = {
    "cartProductResponse": {
        "data": [
            {
                "cartId": "d7e23521-0d23-4f37-aceb-77f8f75b3d16",
                "productId": "32bdd633-932f-465a-b93c-219a70971008",
                "name": "600-311-8291",
                "imageThumbnailUrl": "https://untr-my.sharepoint.com/:i:/g/personal/cst_dev04_unitedtractors_com/EW__sgepd2xJs1m7uxNquHQBbRKmqLZ3DkmR7N1Mb7JWww?e=1f9bRa",
                "qty": 3,
                "priceAfter": 300000,
                "priceBefore": 500000,
                "currency": "IDR",
                "cookiesData": ""
            },
            {
                "cartId": "10d112f9-0a0b-4c26-87b6-87ffb0374a77",
                "productId": "4211034d-8571-4ed6-84b2-8b380443bf7b",
                "name": "568002",
                "imageThumbnailUrl": "https://untr-my.sharepoint.com/:i:/g/personal/cst_dev04_unitedtractors_com/EW__sgepd2xJs1m7uxNquHQBbRKmqLZ3DkmR7N1Mb7JWww?e=1f9bRa",
                "qty": 2,
                "priceAfter": 800000,
                "priceBefore": 1000000,
                "currency": "IDR",
                "cookiesData": ""
            }
        ],
        "meta": {
            "totalItems": 2,
            "pageNumber": 1,
            "pageSize": 5,
            "totalPages": 1,
            "hasPreviousPage": false,
            "hasNextPage": false,
            "nextPageNumber": 1,
            "previousPageNumber": 1
        }
    },
    "subTotalPrice": 2500000
}
