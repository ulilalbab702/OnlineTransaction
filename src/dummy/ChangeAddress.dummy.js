export const dataBillingAddressDummy = [
    {
        "billingId": "34e6045b-5452-409d-cf3b-08d8bb5f8bb8",
        "fullName": "Dwiky Fany Irawan",
        "contactNumber": "081234518768",
        "email": "dwiky@unitedtractors.com",
        "position": "string",
        "addressLabel": "Kantor PT Buana Mining ",
        "city": "Jakarta",
        "address": "Jl. Raya Bekasi No.KM.22, RT.7/RW.1, Cakung Bar., Kec. Cakung, Kota Jakarta Timur, Daerah Khusus Ibukota Jakarta 13910",
        "postalCode": "13910",
        "locationLatitude": "3.669276961596295",
        "locationLongitude": "102.4003031491451",
        "isPrimary": false
    },
    {
        "billingId": "bbc23959-d18a-480e-cf3c-08d8bb5f8bb8",
        "fullName": "Indah Kurnia",
        "contactNumber": "088615246576",
        "email": "indahks@unitedtractors.com",
        "position": "string",
        "addressLabel": "Kantor PT Global Service Indonesia",
        "city": "Jakarta",
        "address": "Jl Raya kaliabang Tengah, Ruko San Jaya, Blok A5 No 9,\r\nSebalah Indomaret",
        "postalCode": "13920",
        "locationLatitude": "3.669276961596295",
        "locationLongitude": "102.4003031491451",
        "isPrimary": true
    },
    {
        "billingId": "3eccd8ca-da3f-469f-cf3d-08d8bb5f8bb8",
        "fullName": "Agit Mahenda",
        "contactNumber": "078126876121",
        "email": "agitmh@unitedtractors.com",
        "position": "string",
        "addressLabel": "Kantor PT Madhani Talatah Nusantara",
        "city": "Semarang",
        "address": "Jl. Raya Randugarut No.Km. 12, Tugurejo, Kec. Tugu, Kota Semarang, Jawa Tengah 50186",
        "postalCode": "50186",
        "locationLatitude": "3.669276961596295",
        "locationLongitude": "102.4003031491451",
        "isPrimary": false
    },
    {
        "billingId": "fafba700-cca2-44b6-cf3e-08d8bb5f8bb8",
        "fullName": "Cuy",
        "contactNumber": "string",
        "email": "string",
        "position": "string",
        "addressLabel": "string",
        "city": "string",
        "address": "string",
        "postalCode": "string",
        "locationLatitude": "string",
        "locationLongitude": "string",
        "isPrimary": false
    },
    {
        "billingId": "45e2a550-91f6-4103-ae40-08d8be27a682",
        "fullName": "hallo guys",
        "contactNumber": "0812",
        "email": "string",
        "position": "string",
        "addressLabel": "kantor",
        "city": "Jakarta",
        "address": "string",
        "postalCode": "string",
        "locationLatitude": "string",
        "locationLongitude": "string",
        "isPrimary": false
    },
    {
        "billingId": "f3acde09-8129-4dd9-5ef3-08d8c0dc1cc9",
        "fullName": "Christiano Messi",
        "contactNumber": "0812",
        "email": "cr7mes@gmail.com",
        "position": "CEO",
        "addressLabel": "Rumah",
        "city": "Palembang",
        "address": "Jalan Mars",
        "postalCode": "333333",
        "locationLatitude": "3.669276961596295",
        "locationLongitude": "102.4003031491451",
        "isPrimary": false
    }
]
