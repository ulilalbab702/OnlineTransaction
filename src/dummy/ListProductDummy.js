export const listProductDummy = {
    "data": [
        {
            "productId": 1,
            "imageUrl": "https://firebasestorage.googleapis.com/v0/b/utportal-d20a4.appspot.com/o/Images%2F37978014-625e-41ff-b29f-639ce3451d42%5C.png?alt=media&token=35f346eb-3199-41e8-a6c6-1fad0a5ee73d",
            "imageThumbnailUrl": "https://firebasestorage.googleapis.com/v0/b/utportal-d20a4.appspot.com/o/Images%2F37978014-625e-41ff-b29f-639ce3451d42%5C.png?alt=media&token=35f346eb-3199-41e8-a6c6-1fad0a5ee73d",
            "serialNumber": "100801",
            "modelNumber": "PC1000SE-1",
            "currency": "IDR",
            "partNumber": "600-311-8293",
            "partNumberDesc": "Cartridge",
            "amount": 20000.0,
            "isAvailable": true,
            "productHierarchy": "KOMAT -COL-",
            "brand": "Komatsu",
            "customerCode": "10611",
            "customerName": "PAMAPERSADA",
            "customerDiscount": 10.0
        },
        {
            "productId": 2,
            "imageUrl": "https://firebasestorage.googleapis.com/v0/b/utportal-d20a4.appspot.com/o/Images%2Fcc9a0189-d1ab-4d8e-a304-8c75ee88fdc6%5C.png?alt=media&token=5fdf1c91-688b-497b-9d2e-6f7a58d0b905",
            "imageThumbnailUrl": "https://firebasestorage.googleapis.com/v0/b/utportal-d20a4.appspot.com/o/Images%2Fcc9a0189-d1ab-4d8e-a304-8c75ee88fdc6%5C.png?alt=media&token=5fdf1c91-688b-497b-9d2e-6f7a58d0b905",
            "serialNumber": "100803",
            "modelNumber": "PC1000SE-1",
            "currency": "IDR",
            "partNumber": "600-311-8293",
            "partNumberDesc": "Cartridge",
            "amount": 20000.0,
            "isAvailable": true,
            "productHierarchy": "KOMAT -COL-",
            "brand": "Komatsu",
            "customerCode": "10611",
            "customerName": "PAMAPERSADA",
            "customerDiscount": 10.0
        },
        {
            "productId": 3,
            "imageUrl": "https://firebasestorage.googleapis.com/v0/b/utportal-d20a4.appspot.com/o/Images%2F7e2fa93a-4485-45af-ba90-e3a5a08333da%5C.png?alt=media&token=448bae01-fac7-4d55-ba87-22d051c5e6b8",
            "imageThumbnailUrl": "https://firebasestorage.googleapis.com/v0/b/utportal-d20a4.appspot.com/o/Images%2F7e2fa93a-4485-45af-ba90-e3a5a08333da%5C.png?alt=media&token=448bae01-fac7-4d55-ba87-22d051c5e6b8",
            "serialNumber": "101930031016",
            "modelNumber": "BC462RB",
            "currency": "IDR",
            "partNumber": "600-311-8293",
            "partNumberDesc": "Cartridge",
            "amount": 20000.0,
            "isAvailable": true,
            "productHierarchy": "BOMAG -COL-",
            "brand": "Bomag",
            "customerCode": "10611",
            "customerName": "PAMAPERSADA",
            "customerDiscount": 10.0
        },
        {
            "productId": 4,
            "imageUrl": "https://firebasestorage.googleapis.com/v0/b/utportal-d20a4.appspot.com/o/Images%2F84921cd8-fcd7-4c1e-816f-6a7d3b9868e6%5C.png?alt=media&token=f66c6b33-cfcb-462e-b8b5-7248af4fbdf1",
            "imageThumbnailUrl": "https://firebasestorage.googleapis.com/v0/b/utportal-d20a4.appspot.com/o/Images%2F84921cd8-fcd7-4c1e-816f-6a7d3b9868e6%5C.png?alt=media&token=f66c6b33-cfcb-462e-b8b5-7248af4fbdf1",
            "serialNumber": "GC5006",
            "modelNumber": "AR-1200M-1",
            "currency": "IDR",
            "partNumber": "600-311-8293",
            "partNumberDesc": "Cartridge",
            "amount": 20000.0,
            "isAvailable": true,
            "productHierarchy": "TADANO -COL-",
            "brand": "Tadano",
            "customerCode": "10611",
            "customerName": "PAMAPERSADA",
            "customerDiscount": 10.0
        },
        {
            "productId": 5,
            "imageUrl": "https://firebasestorage.googleapis.com/v0/b/utportal-d20a4.appspot.com/o/Images%2F116364fd-0839-4fef-9d2e-bc22d39cd3df%5C.png?alt=media&token=a1cc5faa-5c49-4da9-accc-ff2cfd48f041",
            "imageThumbnailUrl": "https://firebasestorage.googleapis.com/v0/b/utportal-d20a4.appspot.com/o/Images%2F116364fd-0839-4fef-9d2e-bc22d39cd3df%5C.png?alt=media&token=a1cc5faa-5c49-4da9-accc-ff2cfd48f041",
            "serialNumber": "JNBCGB45A3AT00123",
            "modelNumber": "CGB45ATHN",
            "currency": "IDR",
            "partNumber": "600-311-8293",
            "partNumberDesc": "Cartridge",
            "amount": 20000.0,
            "isAvailable": true,
            "productHierarchy": "UD -COL-",
            "brand": "UD Trucks",
            "customerCode": "10611",
            "customerName": "PAMAPERSADA",
            "customerDiscount": 10.0
        }
    ],
    "meta": {
        "totalItems": 5,
        "pageNumber": 1,
        "pageSize": 5,
        "totalPages": 1,
        "hasPreviousPage": false,
        "hasNextPage": false,
        "nextPageNumber": 1,
        "previousPageNumber": 1
    }
}