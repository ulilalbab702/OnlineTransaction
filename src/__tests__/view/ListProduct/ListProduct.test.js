import React from "react";
import toJson from "enzyme-to-json";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import ListProductPage from "views/ListProduct/ListProduct";
import { listProductDummy } from "./../../../dummy/ListProductDummy";

Enzyme.configure({ adapter: new Adapter() });
describe("List Product Component", () => {
    const wrapper = shallow(
        <ListProductPage
            listProductData={listProductDummy}
        />
    );
    describe("List Product Component", () => {
        it("Should match to snapshot", () => {
            expect(toJson(wrapper)).toMatchSnapshot();
        })
    })
})