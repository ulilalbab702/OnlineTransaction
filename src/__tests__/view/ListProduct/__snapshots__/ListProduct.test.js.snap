// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List Product Component List Product Component Should match to snapshot 1`] = `
<Col
  className="bgTP"
  style={
    Object {
      "paddingLeft": "0",
    }
  }
  tag="div"
  widths={
    Array [
      "xs",
      "sm",
      "md",
      "lg",
      "xl",
    ]
  }
>
  <Col
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <img
      className="logoBrand"
      src="logo_komatsu.png"
    />
    <div
      className="bannerWelcome"
    >
      <Row
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Col
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <p
            className="textBrand"
          >
            Welcome to Komatsu Parts
          </p>
          <p
            className="textDescBrand"
          >
            Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
          </p>
        </Col>
        <img
          className="imagePC200"
          src="komatsupc200.png"
        />
      </Row>
    </div>
  </Col>
  <Row
    style={
      Object {
        "marginTop": "2rem",
      }
    }
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <div
      style={
        Object {
          "marginRight": "2rem",
          "width": "180px",
        }
      }
    >
      <p
        className="titleFilter1"
      >
        Filter
      </p>
      <hr
        className="lineListProduct"
      />
      <p
        className="titleList"
      >
        Brand
      </p>
      <FormGroup
        check={true}
        tag="div"
      >
        <Label
          check={true}
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Input
            name="radio1"
            type="radio"
          />
           
          <p
            className="titleRadio"
          >
            Komatsu
          </p>
        </Label>
      </FormGroup>
      <FormGroup
        check={true}
        tag="div"
      >
        <Label
          check={true}
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Input
            name="radio1"
            type="radio"
          />
           
          <p
            className="titleRadio"
          >
            UD Truck
          </p>
        </Label>
      </FormGroup>
      <FormGroup
        check={true}
        tag="div"
      >
        <Label
          check={true}
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Input
            name="radio1"
            type="radio"
          />
           
          <p
            className="titleRadio"
          >
            Scania
          </p>
        </Label>
      </FormGroup>
      <FormGroup
        check={true}
        tag="div"
      >
        <Label
          check={true}
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Input
            name="radio1"
            type="radio"
          />
           
          <p
            className="titleRadio"
          >
            Bomag
          </p>
        </Label>
      </FormGroup>
      <FormGroup
        check={true}
        tag="div"
      >
        <Label
          check={true}
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Input
            name="radio1"
            type="radio"
          />
           
          <p
            className="titleRadio"
          >
            Tadano
          </p>
        </Label>
      </FormGroup>
      <hr />
      <p
        className="titleList"
      >
        Category
      </p>
      <FormGroup
        check={true}
        tag="div"
      >
        <Label
          check={true}
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Input
            name="radio2"
            type="radio"
          />
           
          <p
            className="titleRadio"
          >
            Battery
          </p>
        </Label>
      </FormGroup>
      <FormGroup
        check={true}
        tag="div"
      >
        <Label
          check={true}
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Input
            name="radio2"
            type="radio"
          />
           
          <p
            className="titleRadio"
          >
            Filter
          </p>
        </Label>
        <div>
          <select
            className="styleFilterForm"
            onClick={[Function]}
          >
            <option
              hidden={true}
            >
              Select Filter
            </option>
          </select>
          <div
            className="boxSelectedFilter"
          />
        </div>
      </FormGroup>
      <FormGroup
        check={true}
        tag="div"
      >
        <Label
          check={true}
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Input
            name="radio2"
            type="radio"
          />
           
          <p
            className="titleRadio"
          >
            Get & Attachment
          </p>
        </Label>
      </FormGroup>
      <FormGroup
        check={true}
        tag="div"
      >
        <Label
          check={true}
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Input
            name="radio2"
            type="radio"
          />
           
          <p
            className="titleRadio"
          >
            Oil & Cmc
          </p>
        </Label>
      </FormGroup>
      <FormGroup
        check={true}
        tag="div"
      >
        <Label
          check={true}
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Input
            name="radio2"
            type="radio"
          />
           
          <p
            className="titleRadio"
          >
            General Part
          </p>
        </Label>
      </FormGroup>
      <FormGroup
        check={true}
        tag="div"
      >
        <Label
          check={true}
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Input
            name="radio2"
            type="radio"
          />
           
          <p
            className="titleRadio"
          >
            Service Kit
          </p>
        </Label>
      </FormGroup>
      <FormGroup
        check={true}
        tag="div"
      >
        <Label
          check={true}
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Input
            name="radio2"
            type="radio"
          />
           
          <p
            className="titleRadio"
          >
            Tasp
          </p>
        </Label>
      </FormGroup>
      <FormGroup
        check={true}
        tag="div"
      >
        <Label
          check={true}
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Input
            name="radio2"
            type="radio"
          />
           
          <p
            className="titleRadio"
          >
            Hose
          </p>
        </Label>
      </FormGroup>
      <FormGroup
        check={true}
        tag="div"
      >
        <Label
          check={true}
          tag="label"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <Input
            name="radio2"
            type="radio"
          />
           
          <p
            className="titleRadio"
          >
            Under Carriage
          </p>
        </Label>
      </FormGroup>
      <hr />
      <p
        className="titleList"
      >
        Rating
      </p>
      <Row
        style={
          Object {
            "marginBottom": "1rem",
            "marginLeft": "1.5rem",
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Input
          name="checkbox1"
          type="checkbox"
        />
         
        <Row
          style={
            Object {
              "marginLeft": "5px",
              "marginTop": "3px",
            }
          }
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <img
            className="ratingList"
            src="Star.png"
          />
          <img
            className="ratingList"
            src="Star.png"
          />
          <img
            className="ratingList"
            src="Star.png"
          />
          <img
            className="ratingList"
            src="Star.png"
          />
          <img
            className="ratingList"
            src="Star.png"
          />
        </Row>
      </Row>
      <Row
        style={
          Object {
            "marginLeft": "1.5rem",
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Input
          name="checkbox1"
          type="checkbox"
        />
         
        <Row
          style={
            Object {
              "marginLeft": "5px",
              "marginTop": "3px",
            }
          }
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <img
            className="ratingList"
            src="Star.png"
          />
          <img
            className="ratingList"
            src="Star.png"
          />
          <img
            className="ratingList"
            src="Star.png"
          />
          <img
            className="ratingList"
            src="Star.png"
          />
          <img
            className="ratingList"
            src="StarGray.png"
          />
          <p
            className="titleStar"
          >
            Ke Atas
          </p>
        </Row>
      </Row>
      <Row
        style={
          Object {
            "marginLeft": "1.5rem",
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Input
          name="checkbox1"
          type="checkbox"
        />
         
        <Row
          style={
            Object {
              "marginLeft": "5px",
              "marginTop": "3px",
            }
          }
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <img
            className="ratingList"
            src="Star.png"
          />
          <img
            className="ratingList"
            src="Star.png"
          />
          <img
            className="ratingList"
            src="Star.png"
          />
          <img
            className="ratingList"
            src="StarGray.png"
          />
          <img
            className="ratingList"
            src="StarGray.png"
          />
          <p
            className="titleStar"
          >
            Ke Atas
          </p>
        </Row>
      </Row>
      <hr />
      <p
        className="titleList"
      >
        Price
      </p>
      <div
        className="boxPrice"
      >
        <p
          className="rp"
        >
          Rp
        </p>
        <div
          className="line-vertical-list"
        />
        <Input
          id="minPrice"
          name="minPrice"
          placeholder="Minimum Price"
          style={
            Object {
              "border": "0px",
            }
          }
          type="price"
        />
      </div>
      <div
        className="boxPrice"
      >
        <p
          className="rp"
        >
          Rp
        </p>
        <div
          className="line-vertical-list"
        />
        <Input
          id="maxPrice"
          name="maxPrice"
          placeholder="Maximum Price"
          style={
            Object {
              "border": "0px",
            }
          }
          type="price"
        />
      </div>
    </div>
    <div>
      <Row
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <p
          className="titleFilterListActive"
          onClick={[Function]}
        >
          Product List
        </p>
        <p
          className="titleFilterList"
          onClick={[Function]}
        >
          Product Manual Input
        </p>
      </Row>
      <hr
        className="lineProduct"
      />
      <Row
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <p
          className="titleSort"
        >
          Sort by
        </p>
        <Input
          className="sortByInput"
          id="sort"
          name="sort"
          placeholder="bestseller"
          type="sortby"
        />
        <p
          className="titleSort"
        >
          1.198 Items
        </p>
      </Row>
      <div
        style={
          Object {
            "marginBottom": 10,
            "marginTop": 10,
          }
        }
      >
        <Row
          className="gridListProduct  defaultMargin"
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
        >
          <div
            className="cardListLP"
            key="0"
          >
            <Col
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              <div
                className="cardImageLP"
              >
                <img
                  className="imageList"
                  src="https://firebasestorage.googleapis.com/v0/b/utportal-d20a4.appspot.com/o/Images%2F37978014-625e-41ff-b29f-639ce3451d42%5C.png?alt=media&token=35f346eb-3199-41e8-a6c6-1fad0a5ee73d"
                />
              </div>
              <p
                className="titleListProduct"
              >
                Komatsu
              </p>
              <p
                className="titleListProduct2"
              >
                Battery Komatsu K150A
              </p>
              <p
                className="titleListProduct3"
              >
                12.000.000
              </p>
              <p
                className="titleListProduct4"
              >
                10.000.000
              </p>
              <div
                class="row"
              >
                <img
                  src="Star.png"
                  style={
                    Object {
                      "height": 10,
                      "marginLeft": 15,
                      "marginRight": 5,
                      "marginTop": 2,
                      "width": 10,
                    }
                  }
                />
                <p
                  className="titleListProduct5"
                >
                  5.0
                </p>
                <p
                  className="titleListProduct"
                >
                  ( 11 review )
                </p>
              </div>
            </Col>
          </div>
          <div
            className="cardListLP"
            key="1"
          >
            <Col
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              <div
                className="cardImageLP"
              >
                <img
                  className="imageList"
                  src="https://firebasestorage.googleapis.com/v0/b/utportal-d20a4.appspot.com/o/Images%2Fcc9a0189-d1ab-4d8e-a304-8c75ee88fdc6%5C.png?alt=media&token=5fdf1c91-688b-497b-9d2e-6f7a58d0b905"
                />
              </div>
              <p
                className="titleListProduct"
              >
                Komatsu
              </p>
              <p
                className="titleListProduct2"
              >
                Battery Komatsu K150A
              </p>
              <p
                className="titleListProduct3"
              >
                12.000.000
              </p>
              <p
                className="titleListProduct4"
              >
                10.000.000
              </p>
              <div
                class="row"
              >
                <img
                  src="Star.png"
                  style={
                    Object {
                      "height": 10,
                      "marginLeft": 15,
                      "marginRight": 5,
                      "marginTop": 2,
                      "width": 10,
                    }
                  }
                />
                <p
                  className="titleListProduct5"
                >
                  5.0
                </p>
                <p
                  className="titleListProduct"
                >
                  ( 11 review )
                </p>
              </div>
            </Col>
          </div>
          <div
            className="cardListLP"
            key="2"
          >
            <Col
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              <div
                className="cardImageLP"
              >
                <img
                  className="imageList"
                  src="https://firebasestorage.googleapis.com/v0/b/utportal-d20a4.appspot.com/o/Images%2F7e2fa93a-4485-45af-ba90-e3a5a08333da%5C.png?alt=media&token=448bae01-fac7-4d55-ba87-22d051c5e6b8"
                />
              </div>
              <p
                className="titleListProduct"
              >
                Bomag
              </p>
              <p
                className="titleListProduct2"
              >
                Battery Komatsu K150A
              </p>
              <p
                className="titleListProduct3"
              >
                12.000.000
              </p>
              <p
                className="titleListProduct4"
              >
                10.000.000
              </p>
              <div
                class="row"
              >
                <img
                  src="Star.png"
                  style={
                    Object {
                      "height": 10,
                      "marginLeft": 15,
                      "marginRight": 5,
                      "marginTop": 2,
                      "width": 10,
                    }
                  }
                />
                <p
                  className="titleListProduct5"
                >
                  5.0
                </p>
                <p
                  className="titleListProduct"
                >
                  ( 11 review )
                </p>
              </div>
            </Col>
          </div>
          <div
            className="cardListLP"
            key="3"
          >
            <Col
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              <div
                className="cardImageLP"
              >
                <img
                  className="imageList"
                  src="https://firebasestorage.googleapis.com/v0/b/utportal-d20a4.appspot.com/o/Images%2F84921cd8-fcd7-4c1e-816f-6a7d3b9868e6%5C.png?alt=media&token=f66c6b33-cfcb-462e-b8b5-7248af4fbdf1"
                />
              </div>
              <p
                className="titleListProduct"
              >
                Tadano
              </p>
              <p
                className="titleListProduct2"
              >
                Battery Komatsu K150A
              </p>
              <p
                className="titleListProduct3"
              >
                12.000.000
              </p>
              <p
                className="titleListProduct4"
              >
                10.000.000
              </p>
              <div
                class="row"
              >
                <img
                  src="Star.png"
                  style={
                    Object {
                      "height": 10,
                      "marginLeft": 15,
                      "marginRight": 5,
                      "marginTop": 2,
                      "width": 10,
                    }
                  }
                />
                <p
                  className="titleListProduct5"
                >
                  5.0
                </p>
                <p
                  className="titleListProduct"
                >
                  ( 11 review )
                </p>
              </div>
            </Col>
          </div>
          <div
            className="cardListLP"
            key="4"
          >
            <Col
              tag="div"
              widths={
                Array [
                  "xs",
                  "sm",
                  "md",
                  "lg",
                  "xl",
                ]
              }
            >
              <div
                className="cardImageLP"
              >
                <img
                  className="imageList"
                  src="https://firebasestorage.googleapis.com/v0/b/utportal-d20a4.appspot.com/o/Images%2F116364fd-0839-4fef-9d2e-bc22d39cd3df%5C.png?alt=media&token=a1cc5faa-5c49-4da9-accc-ff2cfd48f041"
                />
              </div>
              <p
                className="titleListProduct"
              >
                UD Trucks
              </p>
              <p
                className="titleListProduct2"
              >
                Battery Komatsu K150A
              </p>
              <p
                className="titleListProduct3"
              >
                12.000.000
              </p>
              <p
                className="titleListProduct4"
              >
                10.000.000
              </p>
              <div
                class="row"
              >
                <img
                  src="Star.png"
                  style={
                    Object {
                      "height": 10,
                      "marginLeft": 15,
                      "marginRight": 5,
                      "marginTop": 2,
                      "width": 10,
                    }
                  }
                />
                <p
                  className="titleListProduct5"
                >
                  5.0
                </p>
                <p
                  className="titleListProduct"
                >
                  ( 11 review )
                </p>
              </div>
            </Col>
          </div>
        </Row>
        <div
          style={
            Object {
              "display": "flex",
              "justifyContent": "center",
            }
          }
        >
          <div
            className="cardMore"
          >
            <p
              className="titleButtonMore"
            >
              See More Product
            </p>
          </div>
        </div>
      </div>
    </div>
  </Row>
</Col>
`;
